### PR TITLE
Kathrin/idealo energieeffizienzklasse

### PR DESCRIPTION
**Pull request by khoffmeister**

### CODE REVIEW

File: de/maerkte/idealo/idealo-einrichten.adoc [hunk 1]
```diff
@@ -205,6 +213,111 @@ a| Wenn du das Preisanpassungs-Feature für idealo Direktkauf-Händler:innen nut
...
+
+*_Hinweis:_* Welche Energieeffizienzskala verwendet wird, gibst du über das idealo-Attribut *Version* an.
+
+[cols="1,3"]
+!===
+! *Neue Eigenschaft erstellen*
+! Erstellt eine Eigenschaft des Typs *Text* und verknüpft diese Eigenschaft mit dem idealo-Attribut. Der Wert, der für diese Eigenschaft an den Varianten gespeichert ist, wird an idealo exportiert. +
+*_Tipp:_* Die Eigenschaft wird in der Eigenschaftsgruppe *idealoDE* gespeichert.
+
+! *Vorhandene Eigenschaft verknüpfen*
```
Comment (Strean1337) [inline de/maerkte/idealo/idealo-einrichten.adoc:237]: Would be nice to mention here, that the customer can only select item properties with type text

File: de/maerkte/idealo/idealo-einrichten.adoc [hunk 2]
```diff
@@ -205,6 +213,111 @@ a| Wenn du das Preisanpassungs-Feature für idealo Direktkauf-Händler:innen nut
...
+
+*_Hintergrund:_* Das Spektrum ist abhängig von der Art des Geräts. Zum Beispiel wird für Haushaltswaschmaschinen ein Spektrum von A bis D verwendet. Für Haushaltskühlgeräte reicht das Spektrum aber von A bis G.
+
+[cols="1,3"]
+!===
+! *Neue Eigenschaft erstellen*
+! Erstellt eine Eigenschaft des Typs *Text* und verknüpft diese Eigenschaft mit dem idealo-Attribut. Der Wert, der für diese Eigenschaft an den Varianten gespeichert ist, wird an idealo exportiert. +
+*_Tipp:_* Die Eigenschaft wird in der Eigenschaftsgruppe *idealoDE* gespeichert.
+
+! *Vorhandene Eigenschaft verknüpfen*
```
Comment (Strean1337) [inline de/maerkte/idealo/idealo-einrichten.adoc:263]: same here

File: de/maerkte/idealo/idealo-einrichten.adoc [hunk 3]
```diff
@@ -205,6 +213,111 @@ a| Wenn du das Preisanpassungs-Feature für idealo Direktkauf-Händler:innen nut
...
+*_Beispiel:_* `http://www.link.to/label.jpg` +
+*_Wichtig:_* Achte darauf, dass die Bildauflösung ausreicht, um den QR-Code der Labels zu scannen.
+
+[cols="1,3"]
+!===
+! *Neue Eigenschaft erstellen*
+! Erstellt eine Eigenschaft des Typs *Datei* und verknüpft diese Eigenschaft mit dem idealo-Attribut. Der Wert, der für diese Eigenschaft an den Varianten gespeichert ist, wird an idealo exportiert. +
+*_Tipp:_* Die Eigenschaft wird in der Eigenschaftsgruppe *idealoDE* gespeichert.
+
+! *Vorhandene Eigenschaft verknüpfen*
```
Comment (Strean1337) [inline de/maerkte/idealo/idealo-einrichten.adoc:280]: same here, but type is file

File: de/maerkte/idealo/idealo-einrichten.adoc [hunk 4]
```diff
@@ -205,6 +213,111 @@ a| Wenn du das Preisanpassungs-Feature für idealo Direktkauf-Händler:innen nut
...
+| Mit dieser Eigenschaft exportierst du Links zu PDF-Dateien, die EEK-Datenblätter für kennzeichnungspflichtige Artikel enthalten. +
+*_Beispiel:_* `http://www.example.com/datasheet.pdf`
+
+[cols="1,3"]
+!===
+! *Neue Eigenschaft erstellen*
+! Erstellt eine Eigenschaft des Typs *Datei* und verknüpft diese Eigenschaft mit dem idealo-Attribut. Der Wert, der für diese Eigenschaft an den Varianten gespeichert ist, wird an idealo exportiert. +
+*_Tipp:_* Die Eigenschaft wird in der Eigenschaftsgruppe *idealoDE* gespeichert.
+
+! *Vorhandene Eigenschaft verknüpfen*
```
Comment (Strean1337) [inline de/maerkte/idealo/idealo-einrichten.adoc:296]: same here, file

File: de/maerkte/idealo/idealo-einrichten.adoc [hunk 5]
```diff
@@ -205,6 +213,111 @@ a| Wenn du das Preisanpassungs-Feature für idealo Direktkauf-Händler:innen nut
...
+* `0` = alte Energieeffizienzskala (bis 28. Februar 2021)
+* `1` = neue Energieeffizienzskala (ab 1. März 2021)
+
+[cols="1,3"]
+!===
+! *Neue Eigenschaft erstellen*
+! Erstellt eine Eigenschaft des Typs *Ganze Zahl* und verknüpft diese Eigenschaft mit dem idealo-Attribut. Der Wert, der für diese Eigenschaft an den Varianten gespeichert ist, wird an idealo exportiert. +
+*_Tipp:_* Die Eigenschaft wird in der Eigenschaftsgruppe *idealoDE* gespeichert.
+
+! *Vorhandene Eigenschaft verknüpfen*
```
Comment (Strean1337) [inline de/maerkte/idealo/idealo-einrichten.adoc:316]: type number

**Review by Strean1337 — CHANGES_REQUESTED** [review]

File: de/maerkte/idealo/idealo-einrichten.adoc (hunk 1 above)
Comment (khoffmeister) [reply de/maerkte/idealo/idealo-einrichten.adoc:237]: Could they not select a property of the type Selection with the energy classes as values as an alternative?

**Pull request merged**